code/modules/reagents/reagent_containers/glass.dm

```diff
@@ -279,3 +279,27 @@
 		slot_equipment_priority.Insert(index, slot_head)
 		return
 	return ..()
+
+/obj/item/weapon/reagent_containers/glass/beaker/waterbottle

```

Comment:
This shouldn't be a beaker. This being a beaker comes across as there being an ulterior motive here, especially in conjunction with the fluorine.

This should be a subtype of `drink`.